timesketch/api/v1/resources/scenarios.py

```diff
@@ -684,6 +684,18 @@ def get(self, sketch_id, question_id):
         """
         sketch = Sketch.get_with_acl(sketch_id)
         question = InvestigativeQuestion.get_by_id(question_id)
+        for conclusion in question.conclusions:
+            conclusion.conclusion_events = []
+            for event in conclusion.events:
+                query_string = f"_id:{event.document_id}"
+                result = self.datastore.search(
+                    sketch_id=event.sketch_id,
+                    query_string=query_string,
+                    query_filter={},
+                    query_dsl={},
+                    indices=[event.searchindex.index_name],
+                )
+                conclusion.conclusion_events.append(result['hits']['hits'][0]['_source'])
```

Comment:
If you change it to one search query, you can also just return the whole result dict and let the frontend handle the events.

Reply:
Done.